### DISCUSSION TITLE
Mentions @, Hashtags # autocompletion for Editor

### DISCUSSION THREAD

**Discussion by artemvalieiev** [dotnet/maui#19611] [Show and tell]
Hello

I want to show you my new library I've crafted for .NET MAUI applications. It's a mention library that enhance editor by easily integrating user mentions and tags. Built on top of LinkedIn's [Spyglass](https://github.com/linkedin/Spyglass) and [Hakawai](https://github.com/linkedin/Hakawai/), this library aims to help add support for mentions and autocompletion. I'd love to hear your thoughts, feedback, and any ideas you might have for making it even better!

Currently I am working on adding rich text editing capabilities like bold, italic, underline, strikethrough, and custom links. I want to bring all customization supports that current MAUI span has.
Got a question, feature request, or want to report an issue? Feel free to drop by the [SpeakLink](https://github.com/artemvalieiev/SpeakLink) repo – your input is invaluable!
Thanks a bunch, and I'm looking forward to your input! 😊

![ios](https://github.com/artemvalieiev/SpeakLink/blob/main/gif/ios.gif?raw=true) ![android](https://github.com/artemvalieiev/SpeakLink/blob/main/gif/android.gif?raw=true)

**Comment by WhoIsFishie** [comment]
very nice. hoping to see more updates in the future

**Comment by theken2309** [comment]
this can read HTML ?
